src/components/review/TextItem.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const TextItem = ({ postData }) => {
+  return (
```

Comment:
```suggestion
  const {id, number, ~~~~~} = postData
  return (
```

postData 구조분해할당해서 사용해도 좋겠네요

Reply:
수정했습니다